### ISSUE TITLE
Per Product Shipping

### ISSUE THREAD

**Issue by justingivens** [academe/wc-multiple-packages#14]
I couldn't get this working with [Per Product Shipping](https://www.woothemes.com/products/per-product-shipping/) Extension from Woothemes.

I had some products with Per Product shipping enabled and some not, and the plugin showed now shipping methods available.

**Comment by judgej** [comment]
Yes, this is generally how these shipping packages plugins work - this one included. Each plugin will throw away all shipping packages that have been built up so far, and then replace them with its own.

Merging the resulting packages from multiple plugins together would be a difficult thing to do (and somewhat ambiguous), and would need to be done by ALL shipping packages plugins for it to work, with an agreement on how they would interact to resolve conflicts.

I think you have a choice of using one or the other, as they both "take over" control of the same thing.

**Comment by judgej** [comment]
There may be a way to influence what order the shipping packages run in, so there is some hope, but still not an easy thing to solve.

**Comment by philow3n** [comment]
Per Product Shipping is now in-built into Woo Vendors, making this plugin redundant.  I have Woo Vendors installed and have no settings screen under Settings -> Shipping for this plugin.
